drivers/sensor/mpu9250/mpu9250.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MPU9250_MPU9250_H_
+#define ZEPHYR_DRIVERS_SENSOR_MPU9250_MPU9250_H_
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <sys/util.h>
+#include <zephyr/types.h>
+#include <kernel.h>
```

Comment:
```suggestion
#include <stdint.h>

#include <device.h>
#include <drivers/gpio.h>
#include <drivers/i2c.h>
#include <drivers/sensor.h>
#include <kernel.h>
```

(and cleanup .c file using this header)

Reply:
Changed